### PR TITLE
i18n(zh-cn): Translated get-static-paths-required.mdx

### PR DESCRIPTION
**Pull request by magnum-zx**
#### What kind of changes does this PR include?

- New or updated content
- Translated content

#### Description
translated get-static-paths-required.mdx

### CODE REVIEW

File: src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx [hunk 1]
```diff
@@ -0,0 +1,26 @@
+---
+# NOTE: This file is auto-generated from 'scripts/error-docgen.mjs'
+# Do not make edits to it directly, they will be overwritten.
+# Instead, change this file: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+# Translators, please remove this note and the <DontEditWarning/> component.
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx:5]: There was not need for translations.

File: src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+title: getStaticPaths() function required for dynamic routes.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+import DontEditWarning from '~/components/DontEditWarning.astro'
+
+<DontEditWarning />
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx:13]: `<DontEditWarning />` not need.

File: src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+
+> **GetStaticPathsRequired**： 对于动态路由来说，使用 `getStaticPaths()` 函数是必要的。请你确保从动态路由中 `export` 了一个 `getStaticPaths` 函数。(E03015)
+
+## 哪里出了问题?
+在[静态模式](/zh-cn/core-concepts/routing/#static-ssg-mode)下，所有路由都必须在构建时确定。因此，动态路由必须`export`一个 `getStaticPaths` 函数，来返回将要生成的不同路径。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx:19]: There has a broken link. `routing.mdx` is outdated

File: src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+
+**请参阅：**
+-  [Dynamic Routes](/zh-cn/core-concepts/routing/#dynamic-routes)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx:22]: There is also a need to translate. Broken link too.
```suggestion
-  [动态路由](/zh-cn/core-concepts/routing/#动态路由)
```

File: src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+-  [`getStaticPaths()`](/zh-cn/reference/api-reference/#getstaticpaths)
+-  [Server-side Rendering](/zh-cn/guides/server-side-rendering/)
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx:24]:
```suggestion
-  [服务端渲染](/zh-cn/guides/server-side-rendering/)
```

**Review by liruifengv — CHANGES_REQUESTED** [review]

**Comment by liruifengv** [comment]
Thanks for your first PR to Astro docs. @magnum-zx 
I pointed out some need to fix.

**Comment by magnum-zx** [comment]
@liruifengv This is my first PR to Astro docs, thanks for your review!
do I need to resubmit the PR after fix?

**Comment by liruifengv** [comment]
> @liruifengv This is my first PR to Astro docs, thanks for your review! do I need to resubmit the PR after fix?↳

You can continue to commit to this PR.

**Comment by liruifengv** [comment]
To make review easier, we recommend creating multiple PRs if there are no broken links between pages each other.

`environment-variables.mdx` seems to have some other outdated content. I suggested you create a new PR to update it.

File: src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+title: getStaticPaths() function required for dynamic routes.
+i18nReady: true
+githubURL: https://github.com/withastro/astro/blob/main/packages/astro/src/core/errors/errors-data.ts
+---
+
+
+> **GetStaticPathsRequired**： 对于动态路由来说，使用 `getStaticPaths()` 函数是必要的。请你确保从动态路由中 `export` 了一个 `getStaticPaths` 函数。(E03015)
+
+## 哪里出了问题?
+在[静态模式](/en/core-concepts/routing/#static-ssg-mode)下，所有路由都必须在构建时确定。因此，动态路由必须`export`一个 `getStaticPaths` 函数，来返回将要生成的不同路径。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/reference/errors/get-static-paths-required.mdx:11]: The link should be `zh-cn`. I don't know why check-link ci not working.

Maybe you need to update the `routing` page.

**Comment by magnum-zx** [comment]
That is OK@Yan-Thomas,I was doing other work before, and I will continue Update of the Routing page today.

**Comment by Yan-Thomas** [comment]
That was fast, thanks @magnum-zx! Could I please ask for another review @liruifengv? Thanks in advance 🙌

**Comment by liruifengv** [comment]
> 

I'm coming.

File: src/content/docs/zh-cn/core-concepts/routing.mdx [hunk 7]
```diff
@@ -19,108 +28,90 @@ src/pages/posts/1.md         -> mysite.com/posts/1
...
+参数也可以包含在路径的单独部分中。例如， `src/pages/[lang]/[version]/info.astro` 文件与上面相同的 `getStaticPaths()` 将生成路由 `/en/v1/info` 和 `/fr/v2/info`。
 
-例如：
+📚 了解更多关于 [`getStaticPaths()`](/zh-cn/reference/api-reference/#getstaticpaths) 。
 
-- `pages/post/[...slug].astro` → (`/post/a`, `/post/a/b`, `/post/a/b/c`, etc.)
+#### 剩余参数
 
-匹配的参数将作为查询参数（例子中的 `slug`）传递给页面。
+如果你的URL路由需要更加灵活，你可以在 `.astro` 文件名中使用一个剩余参数（`[...path]`），去匹配任何深度的文件路径。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/core-concepts/routing.mdx:89]: There is a missing link here.

File: src/content/docs/zh-cn/core-concepts/routing.mdx [hunk 8]
```diff
@@ -193,18 +280,18 @@ const { page } = Astro.props;
...
+当你使用 `paginate()` 函数时，每个页面将通过 `page` 传递数据。`page` 有很多有用的属性，下面列出最为重要的：
 
-- **page.data** - 数组，包含你传递给 `paginate()` 函数的页面的数据片段
-- **page.url.next** - 链接到该集合中的下一个页面
-- **page.url.prev** - 链接到集合中的上一个页面
+- **page.data** - 数组，包含你传递给 `paginate()` 函数的页面数据片段
+- **page.url.next** - 下一个页面的链接
+- **page.url.prev** - 上一个页面的链接
 
 ```astro
```
Comment (liruifengv) [inline src/content/docs/zh-cn/core-concepts/routing.mdx:296]: This code snippet is missing highlighting.

File: src/content/docs/zh-cn/core-concepts/routing.mdx [hunk 9]
```diff
@@ -254,19 +341,18 @@ interface Page<T = any> {
 ## 嵌套分页
 
 分页的一个更高级的用例是**嵌套分页**。当分页与其他动态路由参数相结合时，你可以使用嵌套式分页和一些属性或标签来将分页进行分类。
-
 例如，如果你想通过一些标签来分组你的分页的 Markdown 帖子，你可以通过创建一个 `/src/pages/[tag]/[page].astro` 的页面来使用嵌套分页，该页面将匹配以下链接。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/core-concepts/routing.mdx:344]:
```suggestion

例如，如果你想通过一些标签来分组你的分页的 Markdown 帖子，你可以通过创建一个 `/src/pages/[tag]/[page].astro` 的页面来使用嵌套分页，该页面将匹配以下链接。
```

File: src/content/docs/zh-cn/core-concepts/routing.mdx [hunk 10]
```diff
@@ -2,14 +2,23 @@
...
 description: 关于 Astro 路由的介绍。
 ---
+import FileTree from '~/components/FileTree.astro'
 
-Astro 的**路由基于文件**，它根据项目的 `src/pages` 目录中的文件结构来生成你的构建链接。当一个文件被添加到 `src/pages` 目录中，它将自动基于文件名生成与之对应的路由。
+Astro 使用**基于文件的路由**，它根据项目 `src/pages` 目录中的文件结构来生成你的构建链接。
+
+## 页面之间的导航
+
+Astro 使用标准 HTML [`<a>` 元素](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) 在路由之间导航。 没有提供特定框架的 `<Link>` 组件。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/core-concepts/routing.mdx:11]:
```suggestion
Astro 使用标准 HTML [`<a>` 元素](https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/a) 在路由之间导航。 没有提供特定框架的 `<Link>` 组件。
```

**Review by liruifengv — APPROVED** [review]
LGTM!

**Pull request merged**